torch/testing/_internal/distributed/ddp_under_dist_autograd_test.py

```diff
@@ -277,13 +277,13 @@ def get_training_examples():
     idx = 0
     # Every example has another one that has exactly the same features but an
     # opposite value. Therefore, their grads cancel each other in all-reduce.
-    for value in (-1, 1):
+    for value in (-1.0, 1.0):
```

Comment:
I think you want to not change these and just update the multipliers for x and y below.
That way you won't need the `int()` below.

Reply:
Sure! I will do that.